index.html

```diff
@@ -1,300 +1,368 @@
 <!DOCTYPE html>
 <html lang="en">
-
-<head>
-  <meta charset="UTF-8" />
-  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
-  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-  <link href="/assets/CSS/style.css" rel="stylesheet" />
-  <title>Portofolio</title>
-</head>
-<body>
-  <header>
-    <div class="flex-container">
-      <div class="logo">
-        <h2>
-          <a href="#">My Logo</a>
-        </h2>
-        <img src="/assets/images/Icon-Menu.jpg" alt="Icon menu" />
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link href="/assets/CSS/style.css" rel="stylesheet" />
+    <title>Portofolio</title>
+  </head>
+  <body>
+    <header>
+      <div class="flex-container">
+        <nav class="logo">
+          <h2>
+            <a href="#">My Logo</a>
+            <button class="hamburger" id="humburger">
+              <img src="/assets/images/Icon-Menu.jpg" alt="Icon menu" />
+            </button>
+          </h2>
+          <ul>
+            <li>
+              <a href="#">Portofolio</a>
+            </li>
+            <li>
+              <a href="#">About</a>
+            </li>
+            <li>
+              <a href="#">Contact</a>
+            </li>
+          </ul>
+        </nav>
+        <div class="header-wrapper">
+          <div class="text">
+            <h2>I’m Mariane <br />Glad to see you!</h2>
+            <p>
+              Hello I’m a software developer! I can help you build a product ,
+              feature or website Look through some of my work and experience! If
+              you like what you see and have a project you need coded, don’t
+              hestiate to contact me.
+            </p>
+          </div>
+          <div class="social-media">
+            <div class="icon-a">
+              <h5><a href="#">Let's connect</a></h5>
```

Comment:
None of your `Let's Connect` headers match the template design. Please fix them all, I'm marking just this one for brevity.

<img width="893" alt="Screen Shot 2022-06-09 at 10 29 13" src="https://user-images.githubusercontent.com/82426682/172885647-f24b33a1-c786-4e04-9e58-3982210f7825.png">